data/keymaps/i386/hcesar/hcesar.map

```diff
@@ -0,0 +1,101 @@
+# hcesar keyboard, for portuguese language
+# based on keyman Hcesar (Portugal) layout
+# Derived from pt-latin1.map 
+# With fixes for keycode 43 from pmmm@camoes.rnl.ist.utl.pt, 980626.
+# With fixes for keycodes 27 and 43 from Francisco Miguel Pedroso Honório Colaço
+# (fcolaco@wolf.fis.uc.pt).
+#
+charset "iso-8859-1"
+keymaps 0-2,4,6,8,12
+alt_is_meta
+include "qwerty-layout"
+include "linux-with-alt-and-altgr"
+include "euro2.map"
+strings as usual
+compose as usual for "iso-8859-1"
+
+keycode   1 = Escape
+keycode   2 = one              exclam
+keycode   3 = two              quotedbl         at
+	control keycode   3 = nul
+keycode   4 = three            numbersign       sterling
+	control keycode   4 = Escape
+keycode   5 = four             dollar           section
+	control keycode   5 = Control_backslash
+keycode   6 = five             percent
+	control	keycode   6 = Control_bracketright
+keycode   7 = six              ampersand
+	control	keycode   7 = Control_asciicircum
+keycode   8 = seven            slash            braceleft
+	control keycode   8 = Control_underscore
+keycode   9 = eight            parenleft        bracketleft
+	control keycode   9 = Delete
+keycode  10 = nine             parenright       bracketright
+keycode  11 = zero             equal            braceright
+keycode  12 = apostrophe       question         backslash
+	control keycode  12 = Control_underscore
+	alt	keycode  12 = Meta_minus
+keycode  13 = guillemotleft    guillemotright
+	alt	keycode  13 = Meta_equal
+keycode  14 = Delete
+keycode  15 = Tab
+	shift	keycode  15 = Meta_Tab
+keycode  16 = minus            underscore
+keycode  17 = h  
+keycode  18 = c 
+keycode  19 = e 
+keycode  20 = s 
+keycode  21 = a
+keycode  22 = r
+keycode  23 = o
+keycode  24 = p
+keycode  25 = z
+keycode  26 = plus             asterisk         dead_diaeresis
+	control keycode  26 = Escape
+	alt	keycode  26 = Meta_bracketleft
+keycode  27 = dead_acute       dead_grave       apostrophe       grave
+	control keycode  27 = Control_bracketright
+	alt	keycode  27 = Meta_bracketright
+keycode  28 = Return
+	alt	keycode  28 = Meta_Control_m
+keycode  29 = Control
+keycode  30 = backslash        bar
+keycode  31 = q
+keycode  32 = t
+keycode  33 = d
+keycode  34 = i
+keycode  35 = n
+keycode  36 = u
+keycode  37 = l
+keycode  38 = m
+keycode  39 = x     
+	alt	keycode  39 = Meta_semicolon
```

Comment:
Why 39 is `x`, but alt 39 is `M-;`? I don't see a key with `M-x`.

Reply:
my mistake